calculator.go

```diff
@@ -1,13 +1,80 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"errors"
+	"fmt"
+	"math"
+)
+
+// Add takes 2+ numbers and returns the result of adding them together.
+func Add(a, b float64, rest ...float64) float64 {
+	result := a + b
+	for _, item := range rest {
+		result += item
+	}
+	return result
+}
+
+// Subtract takes 2+ numbers and subtracts them together.
+func Subtract(a float64, b float64, rest ...float64) float64 {
+	result := a - b
+	for _, item := range rest {
+		result -= item
+	}
+	return result
+}
+
+// Multiply multiplies 2+ numbers.
+func Multiply(a float64, b float64, rest ...float64) float64 {
+	result := a * b
+	for _, item := range rest {
+		result *= item
+	}
+	return result
+}
+
+// Divide divides 2+ numbers or returns an error if it would divide by 0.
+func Divide(a float64, b float64, rest ...float64) (float64, error) {
+	if b == 0 {
+		return 0, errors.New("can't divide by zero")
+	}
+	result := a / b
+	for _, item := range rest {
+		if item == 0 {
+			return 0, errors.New("can't divide by zero")
+		}
+		result /= item
+	}
+	return result, nil
+}
+
+// Sqrt takes a nonnegative number and returns its square root or returns an error.
```

Comment:
"or returns an error"... under what circumstances? Remember, this is the entire documentation for your function. What you've written here is something like the equivalent of "This function will work, or not."